### PR TITLE
Update dependencies, composer. Move to new version of symfony

### PR DESCRIPTION
**Pull request by vkhramtsov**
- [x] Update dependencies (on old symfony)
- [x]  Update composer
- [x] Switch to new version of symfony
- [x] Fix security checker issue

### CODE REVIEW

**Comment by vkhramtsov** [comment]
@chilic  Please review

File: web/app.php [hunk 1]
```diff
@@ -3,24 +3,15 @@
...
-// You should change the ApcClassLoader first argument to a unique prefix
-// in order to prevent cache key conflicts with other applications
-// also using APC.
-/*
-$apcLoader = new Symfony\Component\ClassLoader\ApcClassLoader(sha1(__FILE__), $loader);
-$loader->unregister();
-$apcLoader->register(true);
-*/
+require __DIR__.'/../vendor/autoload.php';
+if (PHP_VERSION_ID < 70000) {
```
Comment (chilic) [inline web/app.php]: @vkhramtsov Looks okay for me. Just one small question.
What the reason to check php version?
Thank you.

Comment (vkhramtsov) [inline web/app.php]: This file is a part of symfony :), you can consult with this here https://github.com/symfony/symfony-standard/blob/v3.3.2/web/app.php#L7

**Comment by vkhramtsov** [comment]
Would you be so kind to merge this PR? I will deploy new version to hosting next week.

**Pull request merged**